### DISCUSSION TITLE
Update 20241206 - issues

### DISCUSSION THREAD

**Discussion by AndyFx12** [rern/rAudio#1827] [Bug Reports]
There are representation issues with portrait-view on my smartphone since last update:
![Screenshot_20241206-131902](https://github.com/user-attachments/assets/442267fb-05c9-4e78-87be-e6b1dd2de7de)
![Screenshot_20241206-131919](https://github.com/user-attachments/assets/50215637-04ec-4d4d-ae36-2065d9eb1e25)

**Comment by AndyFx12** [comment]
Partially fixed with 202412061 - but not with "Back on left side" (and wrong font color?):
![Screenshot_20241206-191729](https://github.com/user-attachments/assets/a89f4107-6804-4964-adb4-2826cc084b91)

**Comment by AndyFx12** [comment]
Positon of Back-button is fixed with 20241207 - but path is missing at portrait-view:
![grafik](https://github.com/user-attachments/assets/798dfdcb-d6d2-49f3-b412-d420e9f9fb76)

and on Smartphone/Firefox/Android, landscape-view is broken:
![grafik](https://github.com/user-attachments/assets/cc735fa9-2141-4f7b-affa-2e66e762d17c)

On Tablet it's ok, though.

regards, Andy

**Reply by rern** [reply]
Hi Andy,

Please try the UPDATE.

**Reply by AndyFx12** [reply]
Hi rern,
- landscape issue is fixed
- missing path is not really fixed ;-):
![Screenshot_20241207-112251](https://github.com/user-attachments/assets/8eeddf9a-1cf8-470a-acb7-e0d0c20a3261)

**Reply by AndyFx12** [reply]
Fixed with latest changes.
